app/components/chart-container.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2020, Verizon Media.
+ * Licensed under the Apache License, Version 2.0. See accompanying LICENSE file for terms.
+ */
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const DATA_LOOKBACK = 30;
+
+export default class ChartContainerComponent extends Component {
+  @service elide;
+  @tracked records;
+  @tracked showModal;
+  @tracked modalChart;
+
+  //feature flag bar chart
+  showBarChart = false;
+
+  @action
+  fetchData() {
+    const { location } = this.args;
+    if (location) {
+      this.fetchRecords.perform(location, this.lookbackDate);
```

Comment:
Clear out old results before loading new ones
```suggestion
      this.records = undefined;
      this.fetchRecords.perform(location, this.lookbackDate);
```